Silent-Vampires/Welcome-to-Nashville-Silent-Vampires
# Gc parks

# Testing Instructions

git fetch --all
git checkout MB-Silent-Vampires-7
check function is in parks.js & apiResultsPrinter.js

compare results with https://data.nashville.gov/resource/74d7-b74t.json
test to see it generates itinerary results


# Checklist:
